src/scripts/scripts.ts

```diff
@@ -6,6 +6,7 @@ import { modulrMasonryController } from './modules/masonry';
 import { modulrSelectController } from './modules/select';
 import { modulrGrid } from './modules/grid';
 import { modulrScrollTo } from './modules/scroll/scroll';
+import { backToTop } from './modules/backToTop/backToTop';
```

Comment:
please can you name the files as the rest of the modules /{moduleName}/index.ts(x)

in that way the import would be cleaner without repetitions